eng/common/scripts/common.ps1

```diff
@@ -1,13 +1,13 @@
 $global:RepoRoot = Resolve-Path "${PSScriptRoot}..\..\..\.."
-$global:EngDir = Join-Path $global:RepoRoot "eng"
-$global:EngCommonDir = Join-Path $global:EngDir "common"
-$global:EngCommonScriptsDir = Join-Path $global:EngCommonDir "scripts"
-$global:EngScriptsDir = Join-Path $global:EngDir "scripts"
+$global:EngDir = Join-Path $RepoRoot "eng"
```

Comment:
After reading about how PS scopes work I don't think we actually need to explicitly use `global` as a prefix here, assuming everyone dot sources in the common.ps1 file. dot sourcing will pull in any variables to the same scope it was sourced into.

Reply:
Yes, I was also getting some issues with that on the pipeline even though it worked locally. I will go ahead and remove it.